### PR TITLE
Add twitter meta tags

### LINKED ISSUE #39: add twitter meta tags
lifted from mozilla.org:

``` html
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="@mozilla">
    <meta name="twitter:domain" content="mozilla.org">
    <meta name="twitter:app:name:googleplay" content="Firefox">
    <meta name="twitter:app:id:googleplay" content="org.mozilla.firefox">
```

### PR DESCRIPTION
**Pull request by FernandaBernardo**
closes #39

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -8,6 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>Mozilla – The Future of Web Games</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@mozilla">
```
Comment (cvan) [inline public_html/index.html]: I think this should read `games.mozilla.org` - what’s it look like in the Twitter Card Previewer?

Comment (FernandaBernardo) [reply public_html/index.html]: I changed, but in previewer needs the og meta tags to look something

File: public_html/index.html [hunk 2]
```diff
@@ -8,6 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>Mozilla – The Future of Web Games</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="games.mozilla.org">
+    <meta name="twitter:domain" content="mozilla.org">
```
Comment (cvan) [inline public_html/index.html]: it looks like this tag is deprecated now

File: public_html/index.html [hunk 3]
```diff
@@ -8,6 +8,15 @@
...
+    <meta name="twitter:app:name:googleplay" content="Firefox">
```
Comment (cvan) [inline public_html/index.html]: you can remove all these `twitter:app` ones

File: public_html/index.html [hunk 4]
```diff
@@ -8,6 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>Mozilla – The Future of Web Games</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="games.mozilla.org">
```
Comment (cvan) [inline public_html/index.html]: do you want to change this to match the document title?

Comment (FernandaBernardo) [reply public_html/index.html]: done

File: public_html/index.html (hunk 2 above)
Comment (FernandaBernardo) [reply public_html/index.html]: removed!

File: public_html/index.html (hunk 3 above)
Comment (FernandaBernardo) [reply public_html/index.html]: done

**Comment by cvan** [comment]
thanks!

**Pull request merged**